docs/getting-started/1-quick-start.md

```diff
@@ -0,0 +1,88 @@
+<!-- front-matter
+id: quick-start
+title: Quick Start
+hide_title: true
+sidebar_label: Quick Start
+-->
+
+# Quick Start
+
+If you've previously installed gulp globally, run `npm rm --global gulp` before following these instructions. For more information, read this [Sip][sip-article].
+
+### Check for node, npm, and npx
+```sh
+node --version
+```
+![Output: v8.11.1][img-node-version-command]
+```sh
+npm --version
+```
+![Output: 5.6.0][img-npm-version-command]
+```sh
+npx --version
+```
+![Output: 9.7.1][img-npx-version-command]
+
+If they are not installed, follow the instructions [here][node-install].
+
+### Install the gulp command line utility
+```sh
+npm install --global gulp-cli
+```
+
+
+### Create a project directory and navigate into it
+```sh
+npx mkdirp my-project
+```
+```sh
+cd my-project
+```
+
+### Create a package.json file in your project directory
+```sh
+npm init
+```
+
+This will guide you through giving your project a name, version, description, etc.
+
+### Install the gulp package in your devDependencies
+```sh
+npm install --save-dev gulp
+```
+
+### Verify your gulp versions
+```sh
```

Comment:
This would probably need some instructions on what to do with output.

Reply:
I think the assumption was that the output should look like the screenshot otherwise you need to do the steps again but maybe that's a poor assumption to make.